doc/developer-guides/hld/hld-security.rst

```diff
@@ -193,6 +193,50 @@ partners are responsible for image signing, ensuring the key strength
 meets security requirements, and storing the secret RSA private key
 securely.
 
+Guest Secure Boot with OVMF
+---------------------------
+Open Virtual Machine Firmware (OVMF) is an EDK II based project to enable UEFI
+support for virtual machines in virtualized environment. In ACRN, OVMF is
+deployed to launch a user VM, as if the user VM is booted on a machine with
+UEFI firmware.
+
+UEFI Secure Boot defines how a platform's firmware can authenticate a digitally
+signed UEFI image, such as an operating system loader or a UEFI driver stored
+in an option ROM. This provides the capability to ensure that those UEFI images
+are only loaded in an owner-authorized fashion and provides a common means to
+ensure platform's security and integrity over systems running UEFI-based firmware.
+UEFI Secure Boot is already supported by OVMF.
+
+:numref:`security-secure-boot-uefi` shows a Secure Boot overview in UEFI.
+
+.. figure:: images/security-image-secure-boot-uefi.png
+   :width: 500px
+   :align: center
+   :name: security-secure-boot-uefi
+
+   UEFI Secure Boot Overview
+
+UEFI Secure Boot is controlled by a set of UEFI Authenticated Variables which specify
+the UEFI Secure Boot Policy, the platform manufacturer or the platform owner enroll the
+policy objects, which include the n-tuple of keys {PK, KEK, db,dbx} as a step 1.
+During each successive boot, the UEFI secure boot implementation will assess the
+policy in order to verify the signed images that are discovered in a host-bus adapter
+or on a disk. If the images pass policy, then they are invoked.
+
+In ACRN, User VM Secure Boot can be enabled by below steps.
+
+#. Generate keys(PK/KEK/DB) with key generation tool such as Ubuntu KeyGeneration,
+   `PK.der`, `KEK.der` and `db.der` will be enrolled in UEFI BIOS, `db.key` and `db.crt`
```

Comment:
Previously, db.der is used. A little confused about db.key and db.crt, if it is the different format of db, please introduce it before.

Reply:
Looks good to me, Thanks!